com.beeper.Beeper.metainfo.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.beeper.Beeper</id>
+  <name>Beeper</name>
+  <project_license>LicenseRef-proprietary</project_license>
+  <developer_name>Beeper Team</developer_name>
+  <summary>All your chats in one app. Yes, really.</summary>
```

Comment:
Please read the appdata guideline

https://docs.flathub.org/docs/for-app-authors/appdata-guidelines/quality-guidelines#summary

This description still feel like misleading advertising.

Reply:
Would "Chat with your friends from other platforms" work?